src/docs/get-started/install/_path-mac-linux.md

```diff
@@ -44,6 +44,40 @@ whenever you open a new window. For example:
     $ which flutter
     ```
 
+### Update directly (Linux)
+
+In most cases, your distribution may not acquire the path you pass
+through the terminal for future sessions. To maintain permanently, in
+some cases, you will need to change the environment variables file
+directly. Remember that it requires administrator privileges.
+For example:
+
+   1. Determine the directory where you placed the Flutter SDK.
+
+   2. Locate the `etc` directory at root of the system and open `profile` file with privileges.
```

Comment:
at root... => at the root...

system and open `profile` file... => system, and open the `profile` file with root privileges.